### PR TITLE
feat(calendar): Enhance accessibility by adding visually hidden text …

### LINKED ISSUE #939: [Feature Request]: <Enhance Calendar Accessibility>
### Describe the problem or use case

issues with the calendar navigation buttons, where screen readers do not properly recognize them. The aim of this issue is to address this concern by implementing a change.

### Proposed solution

Add Visually Hidden Text to Calendar Navigation Buttons

### Alternatives considered

_No response_

### Additional technical context

The visually hidden text has been implemented using a span element with a class that applies CSS rules to hide it from sighted users while keeping it accessible to screen readers.

### PR DESCRIPTION
**Pull request by mryunt02**
### Changes
- Added visually hidden text to the navigation buttons for improved accessibility:
  - `<span class="visually-hidden">Previous Calendar View</span>` was added to the previous button.
  - `<span class="visually-hidden">Next Calendar View</span>` was added to the next button.

### CSS
- Added the following CSS class to hide the text visually while keeping it accessible to screen readers:
  ```css
  .visually-hidden {
    position: absolute;
    width: 1px;
    height: 1px;
    margin: -1px;
    padding: 0;
    overflow: hidden;
    clip: rect(0, 0, 0, 0);
    border: 0;
  }
### Related Issue
Fixes #939

### CODE REVIEW

**Review by erbilnas — CHANGES_REQUESTED** [review]
Thank you so much for this finding about accessibility 🙏 Setting the `label` attribute for bl-button works simply fine—no need to add an extra class. Example: [Icon-Only Buttons](https://baklava.design/?path=/docs/components-button--documentation#icon-only-buttons)

File: src/components/calendar/bl-calendar.ts [hunk 1]
```diff
@@ -521,7 +521,9 @@ export default class BlCalendar extends LitElement {
             variant="tertiary"
```
Comment (erbilnas) [inline src/components/calendar/bl-calendar.ts:521]: label="Previous Calendar View" for example

**Comment by erbilnas** [comment]
Also, if you could apply the same for other buttons, we’d really appreciate it! 🧡

File: src/components/calendar/bl-calendar.ts (hunk 1 above)
Comment (mryunt02) [reply src/components/calendar/bl-calendar.ts:521]: Oh! I missed the label property of button elements, Thank you. I'm on it

**Review by erbilnas — APPROVED** [review]

File: src/components/calendar/bl-calendar.ts (hunk 1 above)
Comment (AykutSarac) [reply src/components/calendar/bl-calendar.ts:522]: I’d suggest updating it to “next/previous year” if possible, as the buttons specifically navigate between **years**. The current version works as well, though.

Comment (mryunt02) [reply src/components/calendar/bl-calendar.ts:522]: I thought about changing the button label to next/previous year, but can also switch between months with this button. So I thought the most logical is the previous/next calendar view.

Comment (AykutSarac) [reply src/components/calendar/bl-calendar.ts:522]: I think we can make it dynamic in this case depending on the use and state.

Comment (mryunt02) [reply src/components/calendar/bl-calendar.ts:522]: Yes, this make sense. I'm on it.

File: src/components/calendar/bl-calendar.ts [hunk 2]
```diff
@@ -511,13 +511,14 @@ export default class BlCalendar extends LitElement {
     const showMonthSelected =
       this._calendarView === CALENDAR_VIEWS.MONTHS ? "header-text-hover" : "";
     const showYearSelected = this._calendarView === CALENDAR_VIEWS.YEARS ? "header-text-hover" : "";
+    const ButtonLabel = this._calendarView === CALENDAR_VIEWS.DAYS ? "Month" : "Year";
```
Comment (erbilnas) [inline src/components/calendar/bl-calendar.ts:514]: the variables should be camelCase

**Review by erbilnas — APPROVED** [review]

**Review by AykutSarac — APPROVED** [review]

**Pull request merged**